management/src/clusterm/manager/api.go

```diff
@@ -15,8 +15,9 @@ import (
 
 // APIRequest is the general request body expected by clusterm from it's client
 type APIRequest struct {
-	Nodes []string `json:"nodes,omitempty"`
-	Addrs []string `json:"addrs,omitempty"`
+	Nodes     []string `json:"nodes,omitempty"`
+	Addrs     []string `json:"addrs,omitempty"`
+	HostGroup string   `json:"hostgroup"`
```

Comment:
we should add `omitempty` tag for host-group as well, since it is optional for all requests other than Commission.